crates/wasi-provider/src/wasi_runtime.rs

```diff
@@ -156,7 +156,7 @@ impl WasiRuntime {
                         })
                         .expect("status should be able to send");
                     // Converting from anyhow
```

Comment:
Nit: these comments are no longer needed and I actually think you can return `Err(e)` because they are already `anyhow` errors

Reply:
I'll get rid of the comment but the call to `anyhow!` allows us to add the message as context so I'll keep that.